apps/webstore/src/app/pages/auth/profile/profile.component.html

```diff
@@ -197,7 +197,7 @@ <h2 i18n="@@account-info">Account info</h2>
                         i18n="@@invalid-city"
                         data-cy="account-invalid-city"
                     >
-                        The city name is too long.
+                        Invalid city
```

Comment:
The alert should say why it is invalid
```suggestion
                The city name is too long
```